src/test/java/org/ice4j/ice/NetworkUtilsTest.java

```diff
@@ -23,6 +23,40 @@
 
 public class NetworkUtilsTest
 {
+    @Test
+    public void testIpv4StringToBytes()
+    {
+        byte[] addr = NetworkUtils.strToIPv4("1");
+        assertNotNull(addr);
```

Comment:
assertNotNull seems unnecessary since you are checking a non-null value in the next line

Reply:
Agreed thanks, removed these.